packages/plugin-core/src/commands/ConfigureWithUI.ts

```diff
@@ -43,7 +45,7 @@ function getWebviewContent2(opts: { title: string }) {
     </style>
   </head>
   <body>
-    <iframe width="100%" height="100%" src="http://localhost:${port}/workspace/config.html"></iframe>
+    <iframe width="100%" height="100%" src="http://localhost:${port}/vscode/configure?ws=${wsRoot}&port=${port}"></iframe>
```

Comment:
you shouldn't need to modify this since only the below block will execute when you are running this in dev mode. when this is ready, we'll remove the entire if/else statement

```
    if (getStage() === "dev") {
      resp = WebViewUtils.genHTMLForWebView({
        title: "Dendron Config",
        view: DendronWebViewKey.CONFIGURE,
      });
    } else {
```